service/resource/cloudformation/create.go

```diff
@@ -1,7 +1,63 @@
 package cloudformation
 
-import "context"
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awscloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/microerror"
+)
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	stackInput, ok := createChange.(awscloudformation.CreateStackInput)
+	if !ok {
+		return microerror.Mask(fmt.Errorf("unexpected type, expecting %T, got %T", stackInput, createChange))
+	}
+
+	_, err := r.awsClient.CreateStack(&stackInput)
+	if err != nil {
+		return err
+	}
```

Comment:
Super nice here. This is how I think it should be, take the state and apply it, nothing else. FYI one exception that we have so far is to check if the received state is the zero value. In this case we do not apply but log a message. IMO this is also ok.

Reply:
ack